Optik/Classes/ImageViewController.swift

```diff
@@ -30,7 +30,8 @@ internal final class ImageViewController: UIViewController {
         }
     }
     private(set) var imageView: UIImageView?
-
+    var likeButton: UIButton?
```

Comment:
It answers part of my question. I guess we can use it the way it is now, I'd need to dive in the code to understand better how those need to be different. I took note of it and will check it out directly later if need be